public/app/core/logsModel.test.ts

```diff
@@ -1186,6 +1187,43 @@ describe('logs volume', () => {
     datasource = new MockObservableDataSourceApi('loki', [], undefined, 'Error message');
   }
 
+  it('applies correct meta data', async () => {
+    setup(setupMultipleResults);
+
+    await expect(volumeProvider).toEmitValuesWith((received) => {
+      expect(received).toMatchObject([
+        { state: LoadingState.Loading, error: undefined, data: [] },
+        {
+          state: LoadingState.Done,
+          error: undefined,
+          data: [
+            {
+              fields: expect.anything(),
+              meta: {
+                custom: {
+                  targets: [
+                    {
+                      target: 'volume query 1',
+                    },
+                    {
+                      target: 'volume query 2',
+                    },
+                  ],
+                  logsVolumeType: LogsVolumeType.FullRange,
+                  absoluteRange: {
+                    from: 1623888000000,
+                    to: 1623888000000,
+                  },
```

Comment:
super tiny nit: i had a bit of a hard time understanding this and realizing it is hardcoded `2021-06-17 00:00:00`. could we somehow calculate this from whatever we use in the setup function?

Reply:
Updated in 1d604dfc1e8d3f75d07059c29dd17e6b585c9a6e